src/main/flight/gps_rescue.c

```diff
@@ -97,6 +97,16 @@ typedef struct {
     float accMagnitudeAvg;
 } rescueSensorData_s;
 
+typedef struct {
+    uint32_t previousTimeUs;
```

Comment:
While extracting these variables into a struct makes it look cleaner, it unfortunately also changes the scope of of them from function to file. This is undesired, variables should be kept at minimum possible scope. Can you please change this back? (Also, function static variables should be defined at the the top of the function they are defined in, so please move them there.)

Reply:
In the original code, only RescueStart() is called at initialization time, so that's why I had to move the variables to a global scope, then I could easily reset the variables at RescueStart().
In order to move the variables back to the function scope then the functions have to be called at initialization time. The resulting code can be a bit confusing (look at the beginning of updateGPSRescueState()). Also, is it worth to increase the instruction count in order to save the linker a global symbol (which is feature prefixed so very unlikely to collide)?
I've done this change as a separate commit. If you confirm this is preferred I'll squash it.
Otherwise, feel free to suggest another way to implement it.

Reply:
Here: 
https://github.com/TonyBlit/betaflight/commit/453e3b2a84c92ecfb16e806d9127772180168b66